.github/CODEOWNERS

```diff
@@ -48,6 +48,7 @@
 /pkg/sql/tests/rsg_test.go   @cockroachdb/sql-experience
 
 /pkg/sql/catalog/            @cockroachdb/sql-schema
+/pkg/sql/catalog/multiregion @cockroachdb/sql-experience
```

Comment:
make it/add `@cockroachdb/multiregion` :P

Reply:
Done.